.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    //  "--debug-game-backend",
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Randovania Client",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "${workspaceFolder}/randovania/__main__.py",
```

Comment:
Wouldn't the program be the venv's python interpreter?

Reply:
No, it's a launch configuration of type `debugpy`. It starts the file listed in `program` with the workspace's default interpreter.